### ISSUE TITLE
[Blog] Tags can't be listed in a widget

### ISSUE THREAD

**Issue by alexislefebvre** [Victoire/victoire#841]
## Description
When editing a blog page template, we can't add a widget that show tags attached to a blog entry.

## Resolution Ideas
Add the `@VIC\BusinessProperty({"textable"})` annotation to tags in https://github.com/Victoire/victoire/blob/a8c6675f6b47e567e59b7f4e4a7028bffbc15d9d/Bundle/BlogBundle/Entity/Article.php#L99-L105

## Related to

| Bundle         |    --    | --             | --       |
| -------------- |:--------:| --             | --       |
| Analytics      |  | Page           |  |
| Blog           | &#10004;         | Query          |          |
| BusinessEntity |          |Seo             |          |
| BusinessPage   |          | Sitemap        |          |
| Core           |          | Template       |          |
| Criteria       |          | Twig           |          |
| Filter         |          | User           |          |
| Form           |          | ViewReference  |          |
| I18n           |          | Widget         |          |
| Media          |          | WidgetMap      |          |
| UI             |          |                |          |

**Comment by alexislefebvre** [comment]
I added the annotation and had the following code:

```
/**
     * Tags of the article.
     *
     * @ORM\ManyToMany(targetEntity="Tag", inversedBy="articles")
     * @ORM\JoinTable(name="vic_article_tags")
     * @Assert\Valid()
     * @VIC\BusinessProperty("textable")
     */
    private $tags;
```

Then I go to the business template of articles, I add a List widget but it can't select a property:

![capture d ecran de 2017-03-15 17-52-53](https://cloud.githubusercontent.com/assets/2071331/23990182/84cf5560-0a35-11e7-87a3-033fa53f6fe3.png)

**Comment by gregumo** [comment]
Fixed in #842
